src/locales/it.json

```
@@ -23,6 +23,7 @@
     },
     "pediatrician": "pediatra",
     "youth": "giovani",
+    "floatingClinic": "Ambulanta za nepredeljene",
```

```suggestion
    "floatingClinic": "Ambulanta za neopredeljene",
```

---

```suggestion
    "floatingClinic": "Clinica per pazienti senza un medico",
```

V Izoli pa uporabljajo izraz "AMBULATORIO GENERALE PER GLI ASSICURATI SENZA MEDICO DI BASE"
https://www.zd-izola.si/ambulante-in-sluzbe/ambulanta-za-neopredeljene/ambulanta-za-neopredeljene

---

@b100w11 nam lahko pomagaš s tem prevodom?

---

```suggestion
    "floatingClinic": "Ambulatorio per gli persone senza medico",
```

---

```suggestion
    "floatingClinic": "Ambulatorio per gli pazienti senza medico",
```